.github/actionlint.yml

```
@@ -0,0 +1,3 @@
+self-hosted-runner:
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)